runtime/doc/lsp.txt

```diff
@@ -689,6 +710,17 @@ declaration()                                      *vim.lsp.buf.declaration()*
 definition()                                        *vim.lsp.buf.definition()*
                 TODO: Documentation
 
+document_highlight()                        *vim.lsp.buf.document_highlight()*
+                Send request to server to resolve document highlights for the 
+                current text document position. This request can be associated
+                to key mapping or to events such as `CursorHold`, eg:
+>
+vim.api.nvim_command [[autocmd CursorHold <buffer> lua vim.lsp.buf.document_highlight()]]
+vim.api.nvim_command [[autocmd CursorHoldI <buffer> lua vim.lsp.buf.document_highlight()]]
+vim.api.nvim_command [[autocmd CursorMoved <buffer> lua vim.lsp.util.buf_clear_references()]]
```

Comment:
`lsp.util` is a junk-drawer that will be refactored later. It should not be exposed/recommended to the user.